cmd/mothership/mothership/start/options.go

```diff
@@ -29,27 +29,29 @@ type Options struct {
 	AuditLogFile                 string
 	AuditLogTenantID             string
 	StopAfterMigration           bool
+	ReconcilerList               []string
```

Comment:
Maybe better to rename it to `serviceList` or something as it indicates which microservice or which deployment we want to scale